pkgs/by-name/qw/qwertone/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  wrapGAppsHook3,
+  pkg-config,
+  alsa-lib,
+  atk,
+  gtk3,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "qwertone";
+  version = "0.5.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.com";
+    owner = "azymohliad";
+    repo = "qwertone";
+    tag = "v${version}";
```

Comment:
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
  pname = "qwertone";
  version = "0.5.0";

  src = fetchFromGitLab {
    domain = "gitlab.com";
    owner = "azymohliad";
    repo = "qwertone";
    tag = "v${finalAttrs.version}";
```

Reply:
If this really useful? When overriding the version, the whole src and cargoDeps need to be overrided.